wdio.conf.js

```diff
@@ -2,11 +2,14 @@ const os = require('os')
 const path = require('path')
 const { spawn } = require('child_process')
 
+
+
 // keep track of the `tauri-driver` child process
 let tauriDriver
 
+const mode = process.env.MODE
 const application =
-  process.env.E2E_APPLICATION || './src-tauri/target/release/kittycad-modeling'
+  process.env.E2E_APPLICATION || `./src-tauri/target/${mode}/kittycad-modeling`
```

Comment:
Was going to suggest using the env var `E2E_APPLICATION` directly in `ci.yml` but we'll likely switch when we need multiple platforms so all good.

Thanks for looking into this! ⚡